browser/BUILD.gn

```diff
@@ -69,6 +69,12 @@ source_set("browser_process") {
     "geolocation/brave_geolocation_permission_context.h",
     "metrics/metrics_reporting_util.cc",
     "metrics/metrics_reporting_util.h",
+    "ntp_sponsored_images/ntp_sponsored_images_service_factory.cc",
```

Comment:
please move this to a separate BUILD.gn file. We need to group dependencies with the files they belong to and dumping everything in one place makes it impossible to figure out which deps are needed and which are not

Reply:
Ok, will add separate `BUILD.gn`